src/pages/EventAdministration/components/EventFormAdmin.tsx

```diff
@@ -1,20 +1,122 @@
+import { useState } from 'react';
 export type EventFormAdminProps = {
   eventId: number;
 };
 
+export type EventFormEditorProps = {
+  eventId: number;
+  formId: string | null;
+  formType: EventFormType;
+};
+
+type FormTemplatePreviewType = Omit<EventFormEditorProps, 'formId'> & {
+  formtemplate: Form;
+};
+const removeIdsFromForm = (form: Form): FormCreate => {
+  const { id, ...restForm } = form; // eslint-disable-line
+  const newFields: Array<TextFormField | SelectFormField> = [];
+
+  form.fields.forEach((field) => {
+    const { id, ...restField } = field; // eslint-disable-line
+    const newOptions: Array<SelectFormFieldOption> = [];
+
+    if (field.type !== FormFieldType.TEXT_ANSWER) {
+      field.options.forEach((option) => {
+        const { id, ...restOption } = option; // eslint-disable-line
+        newOptions.push(restOption as SelectFormFieldOption);
+      });
+    }
+    newFields.push({ ...restField, options: newOptions } as TextFormField | SelectFormField);
+  });
+
+  return { ...restForm, fields: newFields } as FormCreate;
+};
+
+const FormTemplatePreview = ({ formtemplate, eventId, formType }: FormTemplatePreviewType) => {
+  const deleteForm = useDeleteForm(formtemplate.id || '-');
+  const { register, formState, getValues, control } = useForm<Form['fields']>();
+  const [isEditing, setIsEditing] = useState(false);
+  const [formtemplateTitle, setFormtemplateTitle] = useState(formtemplate.title);
+  const createForm = useCreateForm();
+  const showSnackbar = useSnackbar();
+
+  const onCreateFormFromTemplate = async () => {
+    const updatedTemplate = removeIdsFromForm(formtemplate);
+    const newForm: EventFormCreate = {
+      title: String(eventId),
+      type: formType,
+      event: eventId,
+      resource_type: FormResourceType.EVENT_FORM,
+      fields: updatedTemplate.fields,
+      template: false,
+    };
+    createForm.mutate(newForm, {
+      onSuccess: (data) => {
+        showSnackbar(data.title, 'success');
+      },
+      onError: (e) => {
+        showSnackbar(e.detail, 'error');
+      },
+    });
+  };
+
+  const onDeleteForm = () => {
+    deleteForm.mutate(undefined, {
+      onSuccess: (data) => {
+        showSnackbar(data.detail, 'success');
+      },
+      onError: (e) => {
+        showSnackbar(e.detail, 'error');
+      },
+    });
+  };
+
+  return (
+    <Expand flat header={formtemplate.title} sx={{ mt: 1 }}>
+      {isEditing ? (
+        <>
+          <TextField label='Navn' onChange={(e) => setFormtemplateTitle(e.target.value)} type='text' value={formtemplateTitle} />
+          <FormFieldsEditor form={formtemplate} newTitle={formtemplateTitle} onSave={() => setIsEditing(false)} />
+        </>
+      ) : (
+        <>
+          <FormView control={control} disabled={true} form={formtemplate} formState={formState} getValues={getValues} register={register} />
+          <Stack>
+            <Stack direction={{ xs: 'column', md: 'row' }} spacing={1} sx={{ pb: 1 }}>
```

Comment:
```suggestion
            <Stack direction={{ xs: 'column', md: 'row' }} gap={1}>
```

Også `gap={1}` på Stack over